app/models/conversation.rb

```diff
@@ -185,6 +182,10 @@ def webhook_data
     Conversations::EventDataPresenter.new(self).push_data
   end
 
+  def cached_label_list_array
+    (cached_label_list || '').split(',')
+  end
```

Comment:
Used in 2 places. If there is a better way to do this let me know @sojan-official